<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>516</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="core::MwcToolbar" name="leftTb" native="true"/>
    </item>
    <item>
     <widget class="QFrame" name="mainWindowFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="helpButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/img/HelpBtn.png</normaloff>:/img/HelpBtn.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="btnSpacerLabel1">
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="connectionStatusLabel">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resources.qrc">:/img/StatusFail.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="rightestSpacerLabel">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbiout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionEvents"/>
    <addaction name="separator"/>
    <addaction name="actionVersion"/>
   </widget>
   <widget class="QMenu" name="menuWallet">
    <property name="title">
     <string>Wallet</string>
    </property>
    <addaction name="actionAccount"/>
    <addaction name="actionContacts"/>
    <addaction name="actionConfig"/>
   </widget>
   <widget class="QMenu" name="menuMWC_bonus">
    <property name="title">
     <string>MWC claims</string>
    </property>
    <addaction name="actionAirdrop"/>
    <addaction name="actionHODL"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionNode_status"/>
    <addaction name="actionConnect_to_node"/>
    <addaction name="actionListening_Status"/>
   </widget>
   <widget class="QMenu" name="menuMoney">
    <property name="title">
     <string>Money</string>
    </property>
    <addaction name="actionSend_coins"/>
    <addaction name="actionRecieve_coins"/>
    <addaction name="actionSend_recieve_offline"/>
    <addaction name="actionTransactions"/>
    <addaction name="actionOutputs"/>
   </widget>
   <addaction name="menuWallet"/>
   <addaction name="menuMoney"/>
   <addaction name="menuConnection"/>
   <addaction name="menuMWC_bonus"/>
   <addaction name="menuAbiout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionVersion">
   <property name="icon">
    <iconset>
     <normalon>:/img/mw-logo.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionAccount">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/accounts.png</normaloff>:/img/accounts.png</iconset>
   </property>
   <property name="text">
    <string>Accounts</string>
   </property>
   <property name="toolTip">
    <string>Wallet accounts</string>
   </property>
  </action>
  <action name="actionEvents">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/notifications.png</normaloff>:/img/notifications.png</iconset>
   </property>
   <property name="text">
    <string>Events</string>
   </property>
  </action>
  <action name="actionHODL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/hodl.png</normaloff>:/img/hodl.png</iconset>
   </property>
   <property name="text">
    <string>HODL</string>
   </property>
   <property name="toolTip">
    <string>HODL - hold and get your reward</string>
   </property>
  </action>
  <action name="actionAirdrop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/airdrop.png</normaloff>:/img/airdrop.png</iconset>
   </property>
   <property name="text">
    <string>Airdrop</string>
   </property>
   <property name="toolTip">
    <string>claim MWC Airdrop money</string>
   </property>
  </action>
  <action name="actionNode_status">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/node_status.png</normaloff>:/img/node_status.png</iconset>
   </property>
   <property name="text">
    <string>Node status</string>
   </property>
   <property name="toolTip">
    <string>Connected MWC node status</string>
   </property>
  </action>
  <action name="actionConnect_to_node">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connect to node</string>
   </property>
  </action>
  <action name="actionContacts">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/contacts.png</normaloff>:/img/contacts.png</iconset>
   </property>
   <property name="text">
    <string>Contacts</string>
   </property>
   <property name="toolTip">
    <string>Contact list</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/config.png</normaloff>:/img/config.png</iconset>
   </property>
   <property name="text">
    <string>Config</string>
   </property>
   <property name="toolTip">
    <string>Wallet configuration</string>
   </property>
  </action>
  <action name="actionListening_Status">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/listening.png</normaloff>:/img/listening.png</iconset>
   </property>
   <property name="text">
    <string>Listening</string>
   </property>
   <property name="toolTip">
    <string>Wallet listening status</string>
   </property>
  </action>
  <action name="actionSend_coins">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/send_coins.png</normaloff>:/img/send_coins.png</iconset>
   </property>
   <property name="text">
    <string>Send coints</string>
   </property>
   <property name="toolTip">
    <string>Send MWC coins</string>
   </property>
  </action>
  <action name="actionSend_recieve_offline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/send_coins_file.png</normaloff>:/img/send_coins_file.png</iconset>
   </property>
   <property name="text">
    <string>Send/recieve offline</string>
   </property>
   <property name="toolTip">
    <string>Send/recieve MWC offline</string>
   </property>
  </action>
  <action name="actionTransactions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/transaction.png</normaloff>:/img/transaction.png</iconset>
   </property>
   <property name="text">
    <string>Transactions</string>
   </property>
  </action>
  <action name="actionOutputs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/outputs.png</normaloff>:/img/outputs.png</iconset>
   </property>
   <property name="text">
    <string>Outputs</string>
   </property>
  </action>
  <action name="actionRecieve_coins">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/receive_coins.png</normaloff>:/img/receive_coins.png</iconset>
   </property>
   <property name="text">
    <string>Recieve coins</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>core::MwcToolbar</class>
   <extends>QWidget</extends>
   <header>core/mwctoolbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
